archiva-modules/archiva-web/archiva-rest/archiva-rest-services/src/main/java/org/apache/archiva/rest/services/DefaultRepositoriesService.java

```diff
@@ -482,6 +482,7 @@ public Boolean copyArtifact( ArtifactTransferRequest artifactTransferRequest )
                 "Artifact \'" + artifactTransferRequest.getGroupId() + ":" + artifactTransferRequest.getArtifactId()
                     + ":" + artifactTransferRequest.getVersion() + "\' was successfully deployed to repository \'"
                     + artifactTransferRequest.getTargetRepositoryId() + "\'";
+            log.debug("copyArtifact {}", msg);
```

Comment:
Other option is to remove String msg completely. Or should it be logged as INFO?